addons/hr_holidays/static/src/discuss/core/public_web/discuss_sidebar_categories_patch.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.DiscussSidebarChannel.main" t-inherit-mode="extension">
+        <xpath expr="//span[hasclass('o-mail-DiscussSidebarChannel-itemName')]" position="inside">
+            <div t-if="thread.correspondent?.persona.outOfOfficeDateEndText" class="text-warning small" t-out="thread.correspondent.persona.outOfOfficeDateEndText"/>
```

Comment:
should use partner_id instead of persona whenever possible (guests don't have outOfOfficeDateEndText), but we can adapt it later when we actually want to remove the persona getter

Reply:
Re-pushed with this, with `partner_id?.` instead of `persona.`